miden-lib/asm/miden/asset.masm

```diff
@@ -115,19 +118,3 @@ export.create_non_fungible_asset
     exec.build_non_fungible_asset
     # => [ASSET]
 end
-
-# PROCEDURES COPIED FROM KERNEL (TODO: get rid of this duplication)
-# =================================================================================================
-
-const.FUNGIBLE_ASSET_MAX_AMOUNT=9223372036854775807
-
-#! Returns the maximum amount of a fungible asset.
-#!
-#! Stack: []
-#! Outputs: [fungible_asset_max_amount]
-#!
-#! fungible_asset_max_amount is the maximum amount of a fungible asset.
-export.get_fungible_asset_max_amount
```

Comment:
Same question as above: is there a reason we can't just re-export this procedure from `utils`?